docs/USAGE-GUIDE.md

```diff
@@ -187,33 +187,33 @@ s2n-tls uses pre-made security policies to help avoid common misconfiguration mi
 
 The following chart maps the security policy version to protocol version and ciphersuites supported.
 
-|          version          | SSLv3 | TLS1.0 | TLS1.1 | TLS1.2 | TLS1.3  | AES-CBC | AES-GCM | ChaCha20-Poly1305 | 3DES | RC4 | DHE | ECDHE |
```

Comment:
I'm not sure I agree with your decision to remove SSLv3 and RC4 from this table. We don't want people to select policies that support those bad ciphersuites/TLS versions on accident. Are you having trouble getting your tool to negotiate these ciphersuites/versions?

Reply:
> Are you having trouble getting your tool to negotiate these ciphersuites/versions?

Okay, I spent some more time playing with the legacy provider and I was able to RC4 working, so we now have that column and I added that to this PR.

The SSLv3 thing was a deliberate omission on my part, and would be significantly more painful to add. I see two paths forward for this PR
1. Add SSLv3 column back in manually -> It ruins the whole "automated" generation thing, but my priority is getting the errors corrected so I'm fine with that.
2. Remove the single(?) policy in this table that supports SSLv3 -> this gets into the "what should our security table look like" question that I was trying hard to avoid. But is a convenient solution to the problem 😄

Reply:
Yeah I would remove 20140601 from the table. It's the only one that supports SSLv3 and it's really old anyways.